### PR TITLE
snes9x_libretro: Super NES emulator core for libretro

### PR DESCRIPTION
**Pull request by kwyxz**
Initially called Snes96, Snes9x has been a groundbreaking Super Famicom / Super Nintendo emulator for over 20 years. It is still actively developed and has always managed to find a good balance between accuracy (though BSnes / Higan is now better at it) and performence.

This is the libretro port of Snes9x, now available for Haiku.

### CODE REVIEW

File: games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe [hunk 1]
```diff
@@ -0,0 +1,62 @@
+SUMMARY="A port of Snes9x, a Nintendo Super NES emulator to the libretro API"
+DESCRIPTION="Snes9x is one of the very first and most well-known Super NES \
+emulators. It has been around since 1996, has always managed to remain as \
+portable as can be, and while not the most accurate Super NES emulator on the \
+market, found a great balance between performance and accuracy. This is the \
+libretro port of the development version of Snes9x."
+HOMEPAGE="http://www.snes9x.com"
```
Comment (fbrosson) [inline games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe]:
```diff
-HOMEPAGE="http://www.snes9x.com"
+HOMEPAGE="http://www.snes9x.com/"
```
(Sorry to ask this...)

File: games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+COPYRIGHT="2007-2018 the Snes9x team, the libretro team"
+LICENSE="
+	Snes9x
+	GNU GPL v2
+	GNU LGPL v2.1
+	"
```
Comment (fbrosson) [inline games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe]: I _think_ we prefer this alternate style:
```diff
-LICENSE="
-	Snes9x
+LICENSE="Snes9x
 	GNU GPL v2
-	GNU LGPL v2.1
-	"
+	GNU LGPL v2.1"
```
(I'll check this asap.)

Comment (fbrosson) [reply games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe]: Check done: a few months ago I had run some scripts to count how many recipes were using each style of `LICENSE`, among those which have more than one license.
The results were:
* around 62 recipes with this style:
```bash
LICENSE="
	Snes9x
	GNU GPL v2
	GNU LGPL v2.1
	"
```
* around 170 recipes with this style:
```bash
LICENSE="Snes9x
 	GNU GPL v2
	GNU LGPL v2.1"
```
So I'd recommend using the second one.
(I can update your PR if you wish.)

Comment (kwyxz) [reply games-emulation/snes9x_libretro/snes9x_libretro-1.56_20180607.recipe]: Please do! :)

**Pull request merged**